lib/format-bar.js

```diff
@@ -1,11 +1,26 @@
+let platformSpace = 0;
+if (process.platform === 'win32'){
```

Comment:
please add some information why this is necessary. what's about linux/osx/bsd ?

Reply:
Saw this on progress https://github.com/visionmedia/node-progress/blob/0790207ef077cbfb7ebde24a1dd9895ebf4643e1/lib/node-progress.js#L152, i did not test it, but i think they use it for a reason.

Reply:
I run test on Windows 10 cmd line, the code is not necessary. I can not reproduce why node-progress uses the 1 space correction.

Reply:
Correction, under win7 cmd line it is not necessary, win 10 cmd line need the space!

Reply:
mostly `cmd.exe` and `powershell.exe` shows a different behaviour regarding ansii escape sequences...this also has to be tested within all major version. therefore we should look for another solution...